src/config.js

```diff
@@ -1,7 +1,13 @@
 import { timemap } from './lib'
 
 export default {
-  gsheets: [],
+  gsheets: [
+    {
+      name: 'us2020',
+      id: '1I_pgyTQJlIorTIEHBxw1mM1STn-SrIi66FKYxut61iM',
+      tabs: timemap.default
+    }
+  ],
```

Comment:
The default config for gsheets should remain as an empty list here, as it allows a lower friction setup (prevents having to sign up for a service account, etc).

It would probably make sense to add a gitignored file that is merged with the default config.js, so that we can keep a local repo saturated with the config we need. Does this make sense?

Reply:
Moreover, it seems like the referenced spreadsheet doesn't conform to the new `export_associations` sheet structure. I could get this PR working in tandem with our front-end when the API (and the front-end's `config.js`) referenced the spreadsheet in `data/timamap_data.xlsx`, but not for the US protests spreadsheet.

Reply:
I agree with you @breezycool about having a separate ignored config. Will make the change accordingly